lib/common/issues/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "issues"
+version = "0.0.0"
+authors = [
+    "Andrey Vasnetsov <andrey@vasnetsov.com>",
+    "Qdrant Team <info@qdrant.tech>",
+]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+[dependencies]
```

Comment:
A bunch of those deps version are now defined in the workspace setting https://github.com/qdrant/qdrant/pull/3490/files

Can you use it there as well?